### DISCUSSION TITLE
Prisma Accelerate Billing

### DISCUSSION THREAD

**Discussion by hughyyyy** [prisma/prisma#23942] [Q&A]
### Question

I might not getting this fully, I have prisma acclerate in placed and does that mean every database query now count towards the Accelerate queries? Even when I am not using the cache function, it will still be charging me? Does this mean that once I have prisma accelerate enabled database queries are not free anymore (Except the monthly 60,000 queries)?

I don't mind paying but I just want to clarify on this front.

**Comment by ludralph** [comment]
Hi @hughyyyy 👋 

Thank you for raising this question.

Yes, once you have Prisma Accelerate enabled, all your database queries will count towards the Accelerate usage, even if you are not using the cache function. This is because Prisma Accelerate is not just a caching solution, but also provides [connection pooling](https://www.prisma.io/docs/accelerate/connection-pooling), which is a method of reusing existing database connections, making your database operations more efficient.

Prisma Accelerate is priced based on your usage, and the costs are calculated based on the number of requests and your egress. The Starter, Pro, and Business plans are available, and you can get started for free and only pay for what you need with full control over your costs. The free tier includes 60,000 queries per month.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by dev-737** [reply]
Hey there! I know I’m a year late, but does the 60,000 queries limit still exist in the free tier? According to the [pricing page](https://www.prisma.io/pricing/bring-your-own-database), it mentions 'First 60k operations free.' Could you please clarify whether this means 60k queries in total or per month? Thanks in advance!

**Reply by nurul3101** [reply]
Yes, the free tier limit still exists. You will get 60K operations per month with Accelerate.

If you decide to use Prisma Postgres, the new database offering by Prisma which comes built in with Accelerate, then you will get 100K requests for free per month.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Reply by aliramazon** [reply]
Hi,

I'm on the Prisma Postgres Starter plan and considering upgrading to Pro. I'm seeing conflicting connection limits across your pages:

Landing page (prisma.io/pricing — "Use Prisma Postgres" tab):

Starter: 10 direct / 100 pooled
Pro: 50 direct / 500 pooled
Plan upgrade confirmation page (in dashboard):

Pro: Maximum connection limit: 100
Docs (prisma.io/docs/postgres/database/connection-pooling):

Table shows Starter: 100, Pro: 500
But text below says: "For Prisma Postgres, the connection limit is currently fixed at 10 and cannot be changed"
Pricing page (prisma.io/pricing — Accelerate section):

Starter: 10, Pro: 100
These four sources show different numbers for the same plans. Could you clarify:

What are the actual pooled and direct connection limits for each Prisma Postgres plan?
Why does the upgrade confirmation page show Pro at 100 while the landing page shows 500?
What does "fixed at 10" mean in the docs if the pooled limit is higher?
If we bypass Accelerate and use the direct postgres:// connection string instead of prisma+postgres://, what happens? Do we only get the direct connection limit (10 on Starter)? Is there any pooling at all in that scenario, or just the Prisma ORM's built-in local pool? Are there any downsides or limitations we should be aware of when using the direct connection?
This is blocking my upgrade decision. Thanks.
